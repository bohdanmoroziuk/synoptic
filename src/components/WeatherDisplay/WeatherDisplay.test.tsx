import { render } from '@testing-library/react-native';

import DisplayWeather from './WeatherDisplay';

describe('WeatherDisplay', () => {
  const mockWeather = {
    location: 'Kyiv',
    weather: 'Light Cloud',
    temperature: 18.3,
  };
  
  it('renders correctly', () => {
    const { toJSON } = render(<DisplayWeather weather={mockWeather} />)

    expect(toJSON()).toMatchSnapshot();
  });
});
