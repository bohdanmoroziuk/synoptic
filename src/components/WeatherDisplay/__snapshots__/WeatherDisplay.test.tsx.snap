// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherDisplay renders correctly 1`] = `
<View>
  <Text
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "AvenirNext-Regular",
          "textAlign": "center",
        },
        Object {
          "fontSize": 44,
        },
      ]
    }
  >
    Kyiv
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "AvenirNext-Regular",
          "textAlign": "center",
        },
        Object {
          "fontSize": 18,
        },
      ]
    }
  >
    Light Cloud
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "AvenirNext-Regular",
          "textAlign": "center",
        },
        Object {
          "fontSize": 44,
        },
      ]
    }
  >
    18Â°C
  </Text>
</View>
`;
